ReactSkia/sdk/CurlNetworking.h

```diff
@@ -13,6 +13,7 @@
 #define DEFAULT_MAX_CACHE_EXPIRY_TIME 1800000 // 30mins in seconds 1800000
 #define CURRENT_CACHE_SIZE 10 // in megabyte
 #define TOTAL_MAX_CACHE_SIZE 15 // in megabyte
+#define MAX_URL_REDIRECT 10L // maximum number of redirects allowed
```

Comment:
Any reason why this value is set to 10

Reply:
As discussed with PO, we are configuring max redirect URL is 10 for now.